addons/website/views/snippets/snippets.xml

```diff
@@ -93,6 +93,8 @@
                 <t t-snippet="website.s_split_intro" string="Split Intro" group="intro">
                     <keywords>content, picture, photo, illustration, media, visual, article, story, combination, network, relationships, interactions, connectivity, collaborations</keywords>
                 </t>
+                <t t-snippet="website.s_banner_connected" string="Banner connected" group="intro" label="Parallax">
+                </t>
```

Comment:
Here we could add some keywords. I suggest to use something like: 

`content, picture, photo, connection, cover, shape, background, image, headings, hero, light, experience`

If @lebl-odoo could have a quick look that would be great 😊